drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -126,32 +130,38 @@
             </label>
         </div>
         <div class="row">
-            <label class="h3" for="item-{{id}}-image-description">{{i18n "Image description (should provide sufficient information to place the item even if the image did not load)"}}</label>
-            <textarea id="item-{{id}}-image-description" {{#if imageURL}}required{{/if}}
-                      class="item-image-description">{{ imageDescription }}</textarea>
+            <label class="h3">
+                <span>{{i18n "Image description (should provide sufficient information to place the item even if the image did not load)"}}</span>
+                <textarea {{#if imageURL}}required{{/if}} class="item-image-description">{{ imageDescription }}</textarea>
+            </label>
         </div>
         <div class="row">
-            <label class="h3" for="item-{{id}}-success-feedback">{{i18n "Success Feedback"}}</label>
-            <textarea id="item-{{id}}-success-feedback"
-                      class="success-feedback">{{ feedback.correct }}</textarea>
+            <label class="h3">
+                <span>{{i18n "Success Feedback"}}</span>
+                <textarea class="success-feedback">{{ feedback.correct }}</textarea>
+            </label>
         </div>
         <div class="row">
-            <label class="h3" for="item-{{id}}-error-feedback">{{i18n "Error Feedback"}}</label>
-            <textarea id="item-{{id}}-error-feedback"
-                      class="error-feedback">{{ feedback.incorrect }}</textarea>
+            <label class="h3">
+                <span>{{i18n "Error Feedback"}}</span>
+                <textarea class="error-feedback">{{ feedback.incorrect }}</textarea>
+            </label>
         </div>
         <div class="row advanced-link">
-            <a href="#">{{i18n "Show advanced settings" }}</a>
+            <a>{{i18n "Show advanced settings" }}</a>
         </div>
         <div class="row advanced">
             <label>
-                {{i18n "Preferred width as a percentage of the background image width (or blank for automatic width):"}}
+                <span>
+                  {{i18n "Preferred width as a percentage of the background image width (or blank for automatic width):"}}
```

Comment:
@staubina I changed the way this field is displayed in the last commit to make it more consistent with other fields. The new layout does not need the colon, so I removed it.